CMakeLists.txt

```diff
@@ -71,7 +71,24 @@ if(pugixml_FOUND)
 else()
   message(STATUS "Did not find pugixml, will use submodule instead")
 endif()
-
+find_package(xtensor QUIET NO_SYSTEM_ENVIRONMENT_PATH)
+if(xtensor_FOUND)
+  message(STATUS "Found xtensor: ${xtensor_DIR} (version ${xtensor_VERSION})")
+else()
+  message(STATUS "Did not find xtensor, will use submodule instead")
+endif()
+find_package(xtl QUIET NO_SYSTEM_ENVIRONMENT_PATH)
+if(xtl_FOUND)
+  message(STATUS "Found xtl: ${xtl_DIR} (version ${xtl_VERSION})")
+else()
+  message(STATUS "Did not find xtl, will use submodule instead")
+endif()
```

Comment:
Since xtl is a transitive dependency (OpenMC → xtensor → xtl) I think we should be able to omit the check for xtl altogether:
```suggestion
```